client/src/infographic/Infographic.js

```diff
@@ -305,8 +371,26 @@ class InfoGraph_ extends React.Component {
       footnotes_for: subject,
     }).then( () => {
       if ( shallowEqualObjectsOverKeys({active_bubble_id, subject, level}, this.state, ['subject','active_bubble_id','level']) ){
+        const panel_filter = _.chain(panel_keys)
+          .map(panel_key => tables_for_panel(panel_key, subject.level))
+          .flatten()
+          .uniq()
+          .reduce((result, table_id) => {
+            result[table_id] = true;
+            return result;
+          }, {})
```

Comment:
A pretty benign mutation, because `result` is totally scoped to within this reduce. Maybe cleaner like this thoung:
```
.reduce(
  (result, table_id) => ({
    ...result,
    [table_id]: true
  }),
  {}
)
```

Alternatively replace the reduce with
```
  .map( (table_id) => [table_id, true] )
  .fromPairs()
```
?

Reply:
ah yes yes